### PR TITLE
Hide the location entry if Maps is disabled and user is not an admin

### PR DESCRIPTION
**Pull request by georgehrke**
To reproduce:
- Disable Maps app
- Login as non-admin user
- Click the "Location" menu entry:

It will show you this error message:
F7CC82A7-8F94-4827-B034-2852D480DB71

Only admins can access this page.

Solution:
- Do not display the "Location" menu entry if Maps is disabled and the current user is no admin.

### CODE REVIEW

**Comment by juliushaertl** [comment]
Bundles seem not up to date :wink:

**Comment by georgehrke** [comment]
> Bundles seem not up to date 😉

No idea where that error is coming from. This is branched off directly from master and i just committed all the files `make` changed.

File: src/Photos.vue [hunk 1]
```diff
@@ -71,6 +75,7 @@ export default {
 			svgplaceholder,
 			imgplaceholder,
 			videoplaceholder,
+			showLocationMenuEntry: OC.isUserAdmin() || isMapsInstalled,
```
Comment (ChristophWurst) [inline src/Photos.vue:78]: why not use https://nextcloud.github.io/nextcloud-auth/modules/_user_.html#getcurrentuser, though? :)

Comment (georgehrke) [reply src/Photos.vue:78]: Excellent idea ;)

**Comment by georgehrke** [comment]
The changes to the actual package.json got lost. Let me take care of that.

**Pull request merged**